concourse/tasks/generate-package-version.yaml

```diff
@@ -0,0 +1,22 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/daisy-builder
+    tag: latest
+
+inputs:
+- name: token
+
+outputs:
+- name: package-version
+
+run:
+  path: sh
+  args:
+  - -xc
```

Comment:
this one doesn't have -e ?

Reply:
I don't know what does with "-e" looks like there is no difference.

Reply:
`-e` means 'exit on errors'. you have it on some tasks and not others, just suggesting we be consistent

Reply:
I will add it for consistent

Reply:
it's still not added here, and x is missing in some others. should be -exc for all, and -ec for ones echoing creds

Reply:
I miss to commit this change.

Reply:
This has been addressed in the latest version.